CONTRIBUTING.rst

```diff
@@ -19,66 +21,66 @@ Upvoting Issues
 ===============
 
 An easy way to contribute is to upvote existing issues that are relevant to
-you. This will give us a better idea what's important for you and other users.
+you. This will give us a better idea what is important for you and other users.
 
 Please avoid content-less ``+1`` comments but instead use the emoji reaction to
 upvote with a 👍. This allows people to `sort issues by reaction`_ and doesn't
 spam the maintainers.
 
 
-Tell us about your experience
-=============================
-
-You don't have to create a detailed bug report or request a new feature to make
-a valuable contribution. Telling us about your experience with CrateDB is
-incredibly valuable as well. Please `get in touch`_ with us to tell us what you
-like and don't like about CrateDB.
-
-
 Reporting Issues
 ================
 
 Before you report an issue, please `search the existing issues`_ to make sure
 someone hasn't already reported it.
 
 When reporting a new issue, include as much detail as possible. For some
-Crate.io_ repositories, issue templates have been configured, and you can just
-follow those.
+Crate.io_ repositories, issue templates have been configured.
 
-For repositories without configured issue templates:
+For repositories without configured issue templates, include:
 
-- Say what you did, what happened, and what you expected to happen
+- what you did, what happened, and what you expected to happen
```

Comment:
I would use sentence case for these bullet points (but omit the final full stop)